### DISCUSSION TITLE
is CompreFace python API allows async calls? What is the best way to use it in flask?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#823] [Q&A] [answered]
is CompreFace python API allows async calls?
What is the best way to use it in flask?

**Comment by pospielov** [comment, marked as answer]
I think no, SDK is synchronous only
https://github.com/exadel-inc/compreface-python-sdk/tree/main/webcam_demo
we have a demo made with Python SDK
